### DISCUSSION TITLE
國考電腦化測驗座位 明年突破1.6萬席

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1802] [General]
如在各類考試服務提供線上申請平台，每年降低80％紙本申請及郵寄，而考試通知書、成績結果、及格通知、複查成績已達100％電子化，且電腦化測驗考試今年減少180萬張紙本試題列印量，減少216株樹木砍伐，節省紙張近4座101樓高。

資料來源：https://news.ltn.com.tw/news/life/breakingnews/4506739

我認為電腦化測驗不但可以縮減行政流程，提高試務作業的效率，還可以達成淨零排放及無紙化的綠色環境。
未來或許可以有更多的科目也可以應用電腦化測試，
例如：數學等需要計畫過程之科目，
但就可能就需要有繪圖板讓考生可以進行計算以及將過程寫入電腦試卷！

**Comment by mrweixd** [comment]
科技即是未來，不過繪圖板跟實際手寫還是有點差

**Comment by eric202501** [comment]
如果要將更多科目引入電腦化測試，特別是需要計畫過程的科目，可能需要先進且合適的測驗平台，才能容納繪圖板等工具，讓考生能夠進行計算並展示他們的解題過程，而且所花費的成本也是一個問題。

**Comment by B1nn9** [comment]
電子化確實是個能減少紙張浪費的重要措施。在幾乎人手一台平板的時代，平常上課做筆記或交作業的我們也能逐漸實現無紙化，不僅節省資源也能為地球盡份心力。

**Comment by Ryanlolry** [comment]
這些數據顯示了電腦化測驗在考試流程中帶來的巨大環境效益，不僅降低了紙張使用，也減少了樹木砍伐，符合環境保護的目標。導入電腦化測驗不僅可以提高行政效率，還有助於實現淨零排放和無紙化的綠色環境。未來，擴大電腦化測驗的範疇，像是數學等需要計算過程的科目，或許需要配備繪圖板讓考生進行計算並將過程寫入電腦試卷，這可以是未來科目應用電腦化測試的發展方向之一。

**Comment by chuanmin0317** [comment]
不得不說電腦化的時代真的來了，越來越多的資料可以使用電子版本的來替代，像是電子書相比傳統書籍來說省去了紙張的浪費，和相較平板來說攜帶的不方便性，不過有時我還是喜歡閱讀傳統書籍，畢竟紙的手感是很難替代的

**Comment by FakeWood** [comment]
真是太棒了，從以前就一直覺得很多通知單沒有必要性，單純就是浪費紙而已，現在這樣就可以節省很多不必要的浪費，希望其他政府機構也能跟進。
另外，原來216棵樹可以做180萬張紙，酷ㄟ

**Comment by YANRONG0123** [comment]
我認為這樣的設備，能夠加強人們與科技之間的互動，現在人類已經難以脫離科技的使用了，提早面對學習新知識，更能夠在之後的職場上有所改變，不願意學習的人只會一個接著一個被淘汰。

**Comment by 20050201Shawn** [comment]
我認為雖然能夠減省大量木材資源，但電費也是一個大問題，不確定目前世界各地有沒有足夠的供電量應付這樣的需要，但若能夠發現新能源或是再發展綠能(太陽能、氫能之類的)，這提議在未來發展會很有看頭。
